boards/arm/stm32/b-g474e-dpow1/scripts/Make.defs

```diff
@@ -85,10 +85,6 @@ endif
 # Provide map file needed by the "Memory Allocation" view in Eclipse:
 LDFLAGS += -Map=$(TOPDIR)/NuttX.map --gc-sections
 
-HOSTCC = gcc
-HOSTINCLUDES = -I.
```

Comment:
isn't this "-I." necessary?

Reply:
No, because many boards don't add -I. without any problem, and C compiler search the current directory by default(#include "xxx.h".

Reply:
there might be code doing #include <xxx.h> where xxx.h is in the current directory.
i don't feel strongly either ways though.

Reply:
In this case, the code is better to change #include "xxx.h".